### ISSUE TITLE
Broken auto completion in vscode on instance object

### ISSUE THREAD

**Issue by moritzraho** [adobe/aio-lib-files#24]
```
const files = await filesLib.init() // => autocompletes
files. // => doesn't autocomplete
```
vscode's intellisense uses typescript types which seem to be compatible/inferred from our JSDoc. But there might be some sort of conflict.

**Comment by moritzraho** [comment]
relates to https://github.com/adobe/aio-lib-state/issues/25

**Comment by shazron** [comment]
fixed in #33
